processor/test/routes.test/operations.spec.ts

```diff
@@ -119,69 +119,160 @@ describe('/operations APIs', () => {
         },
       });
     });
+  });
 
-    describe('GET /operations/status', () => {
```

Comment:
The `describe('GET /operations/status', () => {` was inside of the `describe('GET /operations/config', () => {` block. I moved it outside on it's own.